plugin/package.json

```diff
@@ -39,7 +39,7 @@
     "webpack-fix-style-only-entries": "^0.6.1"
   },
   "dependencies": {
-    "jenkins-js-modules": "1.3.0",
+    "ace-builds": "^1.15.0",
```

Comment:
With the removal of the dependency on [this plugin](https://plugins.jenkins.io/ace-editor/), `jenkins-js-modules` is no longer needed.